### DISCUSSION TITLE
How to deal with Bun-specific syntax causing TSC errors?

### DISCUSSION THREAD

**Discussion by davidgomes** [uptrace/bun#1008] [Q&A]
I have this code which runs fine on Bun:

```
import { readFileSync } from 'node:fs';
import packageJson from '../package.json' with { type: 'file' };
```

I believe the `with { type: 'file' }` syntax is Bun-only. This means that when I run `tsc` for type checking in my CI, I get errors. I also get errors when using ESLint.

How can I deal with this?

**Comment by kylemcc** [comment]
@davidgomes I think you are looking for this github repo: https://github.com/oven-sh/bun. This repository is for the Go ORM.
